src/Nokia_LCD.cpp

```diff
@@ -229,14 +229,24 @@ bool Nokia_LCD::printCharacter(char character) {
 bool Nokia_LCD::draw(const unsigned char bitmap[],
                      const unsigned int bitmap_size,
                      const bool read_from_progmem) {
+    
+    return Nokia_LCD::draw(bitmap, bitmap_size, kTotal_columns, read_from_progmem);
+}
+
+bool Nokia_LCD::draw(const unsigned char bitmap[],
+                     const unsigned int bitmap_size,
+                     const unsigned int bitmap_width,
+                     const bool read_from_progmem) {
     bool out_of_bounds = false;
+    const unsigned int currentX = mX_cursor;
     for (unsigned int i = 0; i < bitmap_size; i++) {
         unsigned char pixel =
             read_from_progmem ? pgm_read_byte_near(bitmap + i) : bitmap[i];
         if (mInverted) {
             pixel = ~pixel;
         }
-        out_of_bounds = sendData(pixel) || out_of_bounds;
+        send(pixel, true, false);
```

Comment:
> send(pixel, **true, false**);

Since we start having two boolean arguments next to each other, this becomes hard to read. Let's use some named variables instead.

Reply:
Calling sendData() needs only one boolean :)